pl_examples/basic_examples/profiler_example.py

```diff
@@ -0,0 +1,58 @@
+import torch
+import torchvision
+import torchvision.models as models
+import torchvision.transforms as T
+
+from pytorch_lightning import LightningDataModule, LightningModule, Trainer
+"""
+
+  This will generate 2 traces: one for training_step and one for validation_step.
+  Open Chrome and copy/paste this url: ``chrome://tracing/``.
+  Once Tracing Opens, click on `Load` at the top-right and load one of the generated traces.
+
```

Comment:
```suggestion
This will generate 2 traces: one for `training_step` and one for `validation_step`.
Open Chrome and copy/paste this url: `chrome://tracing/`.
Once tracing opens, click on `Load` at the top-right and load one of the generated traces.
```